tests/docker/config/delta-postgres.conf

```diff
@@ -107,6 +107,11 @@ plugins {
 
       remote-disk {
         enabled = true
+        authentication: {
+          user: "delta"
+          password: "shhh"
+          realm: "internal"
+        }
```

Comment:
This matches what is in the tests. I don't see a way to define the values in a single place, so this will have to do